lib/tasks/sync_kubernetes_secrets.rake

```diff
@@ -4,10 +4,7 @@ namespace :kubernetes do
     client = Kubernetes::Client.new
     config_map = client.get_config_map(args[:config_map_name])
```

Comment:
We don't need to fetch the config map in this case?

Reply:
Ah yup sorry just realised that too! Fixed now I think.